### PR TITLE
Bathroom monitor

### PR DESCRIPTION
**Pull request by Omnihuron**
screen shot 2016-02-19 at 4 17 21 pm

:tada:

### CODE REVIEW

File: app/assets/javascripts/bathroom_monitor.coffee [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Statusboard.BathroomMonitor
+  constructor: ->
+    setInterval(@liveUpdate, 10000)
+    $(@liveUpdate)
+
+  liveUpdate: =>
+    $.ajax { url: 'api/bathroom_updates' }
```
Comment (artfuldodger) [inline app/assets/javascripts/bathroom_monitor.coffee]: Can probably simplify a bit to `$.get('api/bathroom_updates')`

File: db/migrate/20160219182323_add_bathroom_updates.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class AddBathroomUpdates < ActiveRecord::Migration
+  def change
+    create_table :bathroom_updates do |t|
+      t.boolean :occupied
+      t.integer :room
```
Comment (artfuldodger) [inline db/migrate/20160219182323_add_bathroom_updates.rb]: We should give the bathrooms names. I dunno which is 1 and which is 2.

File: app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -321,6 +321,28 @@ h3 {
     font-size: $small-text;
   }
 
+  #bathroom-monitor {
+    float: right;
+    padding-top: 15px;
+    padding-right: $side-margins;
+
+    .green-circle {
```
Comment (artfuldodger) [inline app/assets/stylesheets/static_pages.scss]: Could use `.occupied` and `.unoccupied` or something

**Comment by artfuldodger** [comment]
:+1:

File: app/assets/javascripts/bathroom_monitor.coffee (hunk 1 above)
Comment (Omnihuron) [inline app/assets/javascripts/bathroom_monitor.coffee]: oh nice, thanks!

File: db/migrate/20160219182323_add_bathroom_updates.rb (hunk 2 above)
Comment (Omnihuron) [inline db/migrate/20160219182323_add_bathroom_updates.rb]: haha me neither... I'll try to think of a way to name them

File: app/assets/stylesheets/static_pages.scss (hunk 3 above)
Comment (Omnihuron) [inline app/assets/stylesheets/static_pages.scss]: ooh yeah good call

File: db/migrate/20160219182323_add_bathroom_updates.rb (hunk 2 above)
Comment (Omnihuron) [inline db/migrate/20160219182323_add_bathroom_updates.rb]: What I should really do is create a `Bathroom` object with a `name` attribute and a `has_many` relationship to `BathroomUpdates`

**Comment by Omnihuron** [comment]
Thanks @artfuldodger! Giving myself a second thumb to have this ready for Monday ;)

**Pull request merged**